### ISSUE TITLE
Uncaught (in promise) TypeError: store.dispatch is not a function createLoader.js:12

### ISSUE THREAD

**Issue by rkmax** [michaelcontento/redux-storage#106]
I just started to use it

```js
import thunk from 'redux-thunk'
import * as storage from 'redux-storage'
import createEngine from 'redux-storage-engine-localstorage'

let storageMiddleware = storage.createMiddleware(engine)
let middleware = applyMiddleware(storageMiddleware, thunk)

const appReducer = storage.reducer(combineReducer({...}))

const store = middleware(createStore)(appReducer, {})

const load = storage.createLoader(engine)
load(storage).then(() => console.log('[storage] loaded'))
```

**Comment by michaelcontento** [comment]
The problem here is that you passed the wrong variable into `load()` :smile: 

`storage.createLoader` creates a new loader-function bound to the given engine. So it does know where to load from but not where to dispatch to. This is why you need to pass the target **store** (not **storage**!) as argument every time you want to load from the bound engine.

**TL;DR**: Just replace:
```js
load(storage).then(() => console.log('[storage] loaded'))
```
with:
```js
load(store).then(() => console.log('[storage] loaded'))
```

:sunglasses:

**Comment by rkmax** [comment]
thanks! @michaelcontento
